src/Service/LoggerService.php

```diff
@@ -18,47 +19,6 @@ class LoggerService
     protected $action;
     protected $data;
 
-    public const PMI_AUDIT_PREFIX = 'PMI_AUDIT_';
```

Comment:
This is pretty minor, but I think it might make sense to keep this constant here. The prefix is only needed internally by this class.